snowflake/cmd/createKmsKey_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	sftesting "antrea.io/theia/snowflake/cmd/testing"
+	kmsclienttesting "antrea.io/theia/snowflake/pkg/aws/client/kms/testing"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+	"github.com/aws/aws-sdk-go-v2/service/kms/types"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	creatingKeyLog = "Creating key"
+	createdKeyLog  = "Created key"
+)
+
+func TestCreateKey(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	origLogger := logger
```

Comment:
I don't think it is very useful to validate the log messages in this test

I think this comment may apply to the other tests as well: IMO you want to rely on log verification when you have no other way to determine whether the correct code path is being executed. But in our case, I feel like we can always check the code path by checking the mock methods being called. Still, you can decide whether we should keep these checks or not.